app/templates/courses/courses-view.pug

```diff
@@ -238,27 +238,28 @@ mixin classrooms_content()
          else
           b(data-i18n="[html]courses.learn_without_classroom_blurb")
 
- hr.class-break
- h3(data-i18n="courses.join_class")
- hr.class-break
-
- form#join-class-form.form-inline
-   if view.utils.isCodeCombat
-     - product = coco
-   else
-     - product = ozar
-   .help-block
-    em(data-i18n="courses.ask_teacher_for_code_" + product)
-   .form-group
-     input#class-code-input.form-control(data-i18n="[placeholder]courses.enter_c_code_" + product, placeholder="<Enter Class Code>", value=view.classCode)
-   input#join-class-button.btn.btn-navy(type="submit", data-i18n="[value]courses.join", value="Join")
-
-   if view.state === 'enrolling'
-     .progress.progress-striped.active
-       .progress-bar(style="width: 100%", data-i18n="courses.joining") Joining class
-
-   if view.errorMessage
-     .alert.alert-danger= view.errorMessage
+ if me.isManualClassroomJoinAllowed()
```

Comment:
wow we have weird indentation in this file. `If` is at a space 1 instead of 2/4 

Probably need to fix the mixin: `mixin classrooms_content()` indent